packages/generate/src/genutil.ts

```diff
@@ -4,7 +4,7 @@ import type {
   DirBuilder,
   IdentRef
 } from "./builders";
-import type * as introspect from "edgedb/dist/reflection/queries/types";
+import * as introspect from "edgedb/dist/reflection/queries/types";
```

Comment:
We _do_ actually import non `type`s from `introspect` so... It's possible we might need two lines to import the types and also import the values, but this should do the right thing since the module contains both.